src/components/Button/Button.js

```diff
@@ -23,48 +23,47 @@ const propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
+const NOOP = () => {};
+
 const Button = props => {
   const [isElementDisabled, customOverrides] = useSelector(state => [
     selectors.isElementDisabled(state, props.dataElement),
     selectors.getCustomElementOverrides(state, props.dataElement),
   ]);
 
-  props = { ...props, ...customOverrides };
   const {
+    disable,
```

Comment:
don't forget to update the documentation / guides / dev site in case you haven't done so